src/components/SidePanel/HelpContent/helpContentMap.tsx

```diff
@@ -0,0 +1,731 @@
+                                </Text>,
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>Tasks</Text> - Record a task, and optionally assign it to someone (or yourself!).
+                                </Text>,
+                            ]}
+                        />
+                    </ExpandableHelp>
+
+                    <Text style={[styles.textHeadlineH2, styles.mv4]}>Actions</Text>
+                    <ExpandableHelp
+                        styles={styles}
+                        title="Hover (or long press) on a comment to see additional options, including:"
+                    >
+                        <BulletList
+                            styles={styles}
+                            items={[
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>React</Text> - Throw a ♥️😂🔥 like on anything!
+                                </Text>,
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>Reply in thread</Text> - Go deeper by creating a new chat on any comment.
+                                </Text>,
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>Mark unread</Text> - Flag it for reading later, at your convenience.
+                                </Text>,
+                            ]}
+                        />
+                    </ExpandableHelp>
+
+                    <Text style={[styles.textHeadlineH2, styles.mv4]}>Composer</Text>
+                    <ExpandableHelp
+                        styles={styles}
+                        title="Use the composer at the bottom to write new messages:"
+                    >
+                        <BulletList
+                            styles={styles}
+                            items={[
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>Markdown</Text> - Format text using <Text style={styles.textBold}>bold</Text>,{' '}
+                                    <Text style={styles.textItalic}>italics</Text>, and{' '}
+                                    <TextLink href="https://help.expensify.com/articles/new-expensify/chat/Send-and-format-chat-messages">more</TextLink>.
+                                </Text>,
+                                <Text style={styles.textNormal}>
+                                    <Text style={styles.textBold}>Mention</Text> - Invite or tag anyone in the world to any chat by putting an @ in front of their email address or phone
+                                    number (e.g., <Text style={styles.textBold}>@awong@marslink.web</Text>, or <Text style={styles.textBold}>@415-867-5309</Text>).
```

Comment:
These are dummy email and phone numbers, right? I'm seeing a ton of various matches for the phone number online so I assume, but just double-checking.

Reply:
I think so, we might want to change it 😅